targets/TARGET_STM/us_ticker.c

```diff
@@ -130,8 +131,9 @@ void init_16bit_timer(void)
     __HAL_TIM_DISABLE_IT(&TimMasterHandle, TIM_IT_CC1);
 
     // Used by HAL_GetTick()
+    total_ticks = 0;
```

Comment:
I wonder why any of these need to be initialised here.

And the fact they weren't in the 32-bit case would seem to confirm that.